src/ngraph_builder.cc

```diff
@@ -1491,6 +1491,25 @@ static Status TranslateFusedBatchNormGradOp(
   return Status::OK();
 }
 
+static Status TranslateSigmoidGradOp(
+    const Node* op, const std::vector<const Tensor*>& static_input_map,
+    Builder::OpMap& ng_op_map) {
+  shared_ptr<ng::Node> ng_input;
+  shared_ptr<ng::Node> ng_delta;
+  TF_RETURN_IF_ERROR(GetInputNodes(ng_op_map, op, &ng_input, &ng_delta));
+
+  auto ng_mul = std::make_shared<ng::op::Multiply>(ng_input, ng_delta);
+  ng::Shape input_shape = ng_input->get_shape();
+  std::vector<std::string> constant_values(ng::shape_size(input_shape), "1");
+  auto c = make_shared<ng::op::Constant>(ng_input->get_element_type(),
+                                         input_shape, constant_values);
+  auto ng_subtract = std::make_shared<ng::op::Subtract>(c, ng_input);
```

Comment:
`c-ng_input` should work.

If broadcast is supported (I think it is), this might just work:
`auto ng_subtract = make_shared<ng::op::Constant>(ng_input->get_element_type(), ng::Shape(1), std::vector<int>({1})) - ng_input;`